test/e2e/ps_test.go

```diff
@@ -405,6 +405,19 @@ var _ = Describe("Podman ps", func() {
 		Expect(actual).ToNot(ContainSubstring("NAMES"))
 	})
 
+	It("podman ps filter by container command", func() {
+		session := podmanTest.Podman([]string{"run", "-d", "--name", "test1", ALPINE, "top"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(ExitCleanly())
```

Comment:
Can you add a second container with a command that doesn't match?

Reply:
Thank you, it sounds reasonable! Test has been added.